src/ts/paging/paging.ts

```diff
@@ -1,14 +1,16 @@
 import { removeChildren } from "../helpers/dom";
-import {OnPagingCb} from "../typing/paging";
-import {WaterfallData, WaterfallDocs} from "../typing/waterfall";
+import { OnPagingCb } from "../typing/paging";
+import { WaterfallData, WaterfallDocs } from "../typing/waterfall";
 
 /** Class to keep track of run of a multi-run har is beeing shown  */
 export default class Paging {
-  private selectedPageIndex = 0;
   private onPageUpdateCbs: OnPagingCb[] = [];
 
-  constructor(private doc: WaterfallDocs) {
-
+  constructor(private doc: WaterfallDocs, private selectedPageIndex = 0) {
+    if (selectedPageIndex >= this.doc.pages.length) {
+      // fall back to last item if doc has too few pages.
+      this.selectedPageIndex = this.doc.pages.length - 1;
```

Comment:
Fall back to last item if doc has too few pages.

I think we should not throw an error for this - this is most likely what the user wanted anyway.
What do you think?